bzip2.rs

```diff
@@ -2050,13 +2050,11 @@ unsafe fn main_0(program_path: &Path) -> IntNative {
     exitValue = 0;
 
     signal(
-        11 as libc::c_int,
-        mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize,
-    );
-    signal(
-        7 as libc::c_int,
+        11,
         mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize,
     );
+    signal(7, mySIGSEGVorSIGBUScatcher as unsafe fn(IntNative) as usize);
```

Comment:
These should be `extern "C"`.

Reply:
yes, though we might be able to remove these entirely ultimately?